### DISCUSSION TITLE
Mailing doesn't work with yarn v3 PnP

### DISCUSSION THREAD

**Discussion by ianwalter** [sofn-xyz/mailing#139] [General]
Just a heads up that Mailing doesn't work with the default yarn v3 config and I had to add the following config settings to the `.yarnrc.yml`:

```yml
nmMode: hardlinks-local
nodeLinker: node-modules
```

**Comment by psugihara** [comment]
Dang, thanks for the note @ianwalter. What error were you seeing? Unable to find required modules?

**Comment by ianwalter** [comment]
The error was essentially:
```
Error: mjml-react tried to access html-minifier, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```
